package.json

```diff
@@ -46,6 +46,35 @@
           "import/first": "off",
           "import/no-duplicates": "off"
         }
+      },
+      {
```

Comment:
The [terra-dev-site overrides are already provided by `@cerner/terra`](https://github.com/cerner/terra-toolkit/blob/main/packages/eslint-config-terra/eslint.config.js#L82) so these can be removed here

Reply:
removed: 654119b0c223845b50ad7ef300a9f732f12bf34a